app.js

```
@@ -23,6 +25,33 @@ var exit = function(status) {
+        result = f(error);
+    } catch (e) {
+        error = e;
+        if (typeof fallback === "function") {
+            result = optional(fallback, defaultValue, finalizer, null, error); // call fallback with the error
+        }
+    } finally {
+        if (typeof finalizer === "function") {
+            try {
+                finalizer(result, error);
+            } catch (e) {
+                return result; // ignore the error in finalizer
```

Biome (2.5.0)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)